### PR TITLE
Fix db in which Ansible remediation sets banner_enable

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Ansible remediation was configuring different db than OVAL checks

#### Notes:

- Caught by test scenario!

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Rule dconf_gnome_banner_enabled:
 Ansible remediation changed.

**Recommended tests to execute:**
 build_product fedora
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-fedora-ds.xml dconf_gnome_banner_enabled

File: linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/ansible/shared.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # disruption = medium
 - name: "Enable GNOME3 Login Warning Banner"
   ini_file:
-    dest: "/etc/dconf/db/local.d/00-security-settings"
+    dest: "/etc/dconf/db/gdm.d/00-security-settings"
```
Comment (matusmarhefka) [inline linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/ansible/shared.yml:8]: This needs to be changed also for the lock from `/etc/dconf/db/local.d/locks/00-security-settings-lock` to `/etc/dconf/db/gdm.d/locks/00-security-settings-lock`.

Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/ansible/shared.yml:8]: Nice catch, fixed by rebase.

**Comment by mildas** [comment]
**Changes identified:**
Rule dconf_gnome_banner_enabled:
 Ansible remediation changed.

**Recommended tests to execute:**
 build_product ol7
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml dconf_gnome_banner_enabled

**Comment by matusmarhefka** [comment]
@yuumasato So when running SSGTS with Ansible remediation there is an error:

```
python3 test_suite.py rule --libvirt qemu:///session rhel82_ssgts --mode online --remediate-using ansible --datastream ../build/ssg-rhel8-ds.xml --profile cis dconf_gnome_banner_enabled
...
INFO - xccdf_org.ssgproject.content_rule_dconf_gnome_banner_enabled
INFO - Script correct_value.pass.sh using profile xccdf_org.ssgproject.content_profile_cis OK
INFO - Script wrong_value.fail.sh using profile xccdf_org.ssgproject.content_profile_cis OK
ERROR - Rule evaluation resulted in fail, instead of expected pass during final stage 
ERROR - The check after remediation failed for rule 'xccdf_org.ssgproject.content_rule_dconf_gnome_banner_enabled'.
```

When using Bash remediation I get no errors. For Ansible the `test_prevent_user_banner_gui_enabled_change` evaluates as true but the `test_banner_gui_enabled`  evaluates as false:
```
I: oscap:   Evaluating textfilecontent54 test 'oval:ssg-test_banner_gui_enabled:tst:1': GUI banner is enabled. [oscap(16662):oscap(7fcf21714d40):oval_resultTest.c:1129:oval_result_test_eval]
I: oscap:     Querying textfilecontent54 object 'oval:ssg-obj_banner_gui_enabled:obj:1', flags: 0. [oscap(16662):oscap(7fcf21714d40):oval_probe.c:113:oval_probe_query_object]
I: oscap:     Creating new syschar for textfilecontent54_object 'oval:ssg-obj_banner_gui_enabled:obj:1'. [oscap(16662):oscap(7fcf21714d40):oval_probe.c:136:oval_probe_query_object]
I: oscap:     I will run textfilecontent54_probe_main: [oscap(16662):probe_worker(7fcecaff5700):worker.c:1080:probe_worker]
I: oscap:     Opening file '/etc/dconf/db/gdm.d/'. [oscap(16662):probe_worker(7fcecaff5700):oval_fts.c:838:oval_fts_open_prefixed]
I: oscap:     Test 'oval:ssg-test_banner_gui_enabled:tst:1' requires that every object defined by 'oval:ssg-obj_banner_gui_enabled:obj:1' exists on the system. [oscap(16662):oscap(7fcf21714d40):oval_resultTest.c:884:_oval_result_test_evaluate_items]
I: oscap:     0 objects defined by 'oval:ssg-obj_banner_gui_enabled:obj:1' exist on the system. [oscap(16662):oscap(7fcf21714d40):oval_resultTest.c:902:_oval_result_test_evaluate_items]
I: oscap:     Test 'oval:ssg-test_banner_gui_enabled:tst:1' does not contain any state to compare object with. [oscap(16662):oscap(7fcf21714d40):oval_resultTest.c:904:_oval_result_test_evaluate_items]
I: oscap:     No item matching object 'oval:ssg-obj_banner_gui_enabled:obj:1' was found on the system. (flag=does not exist) [oscap(16662):oscap(7fcf21714d40):oval_resultTest.c:938:_oval_result_test_evaluate_items]
I: oscap:   Test 'oval:ssg-test_banner_gui_enabled:tst:1' evaluated as false. [oscap(16662):oscap(7fcf21714d40):oval_resultTest.c:1148:oval_result_test_eval]
```

**Comment by mildas** [comment]
**Changes identified:**
Rule dconf_gnome_banner_enabled:
 Ansible remediation changed.

**Recommended tests to execute:**
 build_product ol7
 test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate ansible --datastream build/ssg-ol7-ds.xml dconf_gnome_banner_enabled

**Comment by yuumasato** [comment]
@matusmarhefka should be fixed now. :)

**Comment by matusmarhefka** [comment]
> @matusmarhefka should be fixed now. :)

Confirmed:
```
$ python3 test_suite.py rule --libvirt qemu:///session rhel82_ssgts --mode online --remediate-using ansible --datastream ../build/ssg-rhel8-ds.xml --profile cis dconf_gnome_banner_enabled
...
INFO - xccdf_org.ssgproject.content_rule_dconf_gnome_banner_enabled
INFO - Script correct_value.pass.sh using profile xccdf_org.ssgproject.content_profile_cis OK
INFO - Script wrong_value.fail.sh using profile xccdf_org.ssgproject.content_profile_cis OK
```

**Pull request merged**